keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -638,24 +640,107 @@ always dispatched immediately.  Following is how the other requests
 are dispatched at a given apiserver.
 
 The concurrency limit of an apiserver is divided among the non-exempt
-priority levels in proportion to their assured concurrency shares.
-This produces the assured concurrency value (ACV) for each non-exempt
-priority level:
+priority levels, and higher ones can do a limited amount of borrowing
+from lower ones.
```

Comment:
Why are we borrowing this way, rather than dividing up the unused concurrency by the shares we already know about?

Reply:
Simplicity.

Reply:
I will try to read this design in more detail, but my current impression is that it is not very simple.

Reply:
I prefer the design in this KEP as well.  I could be persuaded to take a different approach if the sharing ended up significantly more effective using the PID controller design, but the design in this KEP appears more easily predictable and debuggable from the outside.